### ISSUE TITLE
Update song, artist, or album information

### ISSUE THREAD

**Issue by bichomen** [Sonerezh/sonerezh#310]
How can I update information, images, names, etc. with Sonerezh? or can not be?

**Comment by gs11** [comment]
Currently the only way is to remove & re-add the folders to have the metadata re-read again. Alternatively click **Reset the database** in the settings.

**Comment by MightyCreak** [comment]
To remove the album in the database, here is [the MySQL line I use in phpMyAdmin](https://github.com/Sonerezh/sonerezh/issues/127#issuecomment-269245161):

    DELETE FROM `songs` WHERE `songs`.`album` = "your album title"

**Comment by MightyCreak** [comment]
@gs11 I think this would be a nice evolution of your PR #293: detect files that have changed (file modification date greater than last update date in the table) and re-read them.

(#293 is good the way it is, small and easy to merge, but maybe another PR sitting on top of it?)

**Comment by MightyCreak** [comment]
Anyhow, I'd say that this particular issue is a dup of #127 and #129

**Comment by gs11** [comment]
@bichomen Feel free to try out an implementation in https://github.com/gs11/sonerezh/commit/26e3b3d1d915f0460278b12313e9cf672357aa95

**Comment by bichomen** [comment]
Hi, thanks for replying, but I see that changing the metadata of a file can only be done using external software like EasyTAG and then upload the modified file to sonerezh.

**Comment by gs11** [comment]
That's correct, the project developers mentioned planning this feature in issue #127.

**Comment by MightyCreak** [comment]
@bichomen I think editing your own files is the best way to go. If it was possible to edit the metadata directly in the Sonerezh database, then obviously, the metadata in the database would start to be different than the metadata in your files. Meaning that if you use the same files in another player, you won't have the same metadata displayed as in Sonerezh. That was what iTunes was doing before, and I think it fundamentally a bad idea.

I advise you to synchronize your local song files on your server (using Nextcloud, rsync, or something else). This way you will, firstly, backup your songs, and secondly, easily be able to edit your files locally and they will synchronize automatically on your server. Personally I use Nextcloud and it works very well; and now with the commit from @gs11 I won't have to delete entries in the songs table so that update the update pass will detect them again, I will just have to hit 'update' and that's it!

**Comment by bichomen** [comment]
@MightyCreak How to synchronize the song folder with nextcloud? I do not see the option in nextcloud

**Comment by MightyCreak** [comment]
I simply upload my music in the music folder of Nextcloud and in Sonerezh I change the music folder in the settings to point to the Nextcloud music folder. And that's it! ;)

**Comment by bichomen** [comment]
What is the problem? The permissions are identical

Current path:
/srv/www/musica/Musica

Path where NextCloud:
/srv/www/nc/data/bichomen/files/Musica

![Permissions](https://user-images.githubusercontent.com/20316285/26929283-59c12b80-4c59-11e7-9874-c945e90938fa.png)
![Error Sonerezh](https://user-images.githubusercontent.com/20316285/26929296-62d94b3a-4c59-11e7-9449-8333276bc679.png)

**Comment by MightyCreak** [comment]
That's weird... If you're on Fedora, I'd search around SELinux maybe?

**Comment by bichomen** [comment]
Not, I'm using Trisquel based in Ubuntu

**Comment by bichomen** [comment]
I was doing tests and it does not accept any folder that is not in the route /srv/www/musica, not even creating symbolic links it works

**Comment by bichomen** [comment]
In the nginx configuration I have:
	root /srv/www/musica/app/webroot;

**Comment by gs11** [comment]
Could it be that your webserver is jailed to www root?

**Comment by bichomen** [comment]
www is bichomen, bichomen is the user of nginx:

`namei -l /srv/www
f: /srv/www
drwxr-xr-x root     root /
drwxr-xr-x root     root srv
drwxr-xr-x bichomen ftp  www
`

**Comment by bichomen** [comment]
In the PHP configuration modifying:

php_admin_value[open_basedir] = /srv/www/musica/:/tmp/

by:

/srv/www/musica/:/home/Musica/

Take the new route, but do not update the list of new songs
